modules/command/common/export-worker-command.js

```diff
@@ -0,0 +1,162 @@
+const Command = require('../command');
+const Models = require('../../../models');
+const { fork } = require('child_process');
+const Utilities = require('../../Utilities');
+const fs = require('fs');
+const path = require('path');
+/**
+ * Increases approval for Bidding contract on blockchain
+ */
+class ExportWorkerCommand extends Command {
+    constructor(ctx) {
+        super(ctx);
+        this.logger = ctx.logger;
+        this.notifyError = ctx.notifyError;
+        this.importService = ctx.importService;
+        this.config = ctx.config;
+        this.web3 = ctx.web3;
+    }
+
+    async execute(command) {
+        const {
+            datasetId,
+            handlerId,
+            standardId,
+        } = command.data;
+
+        const forked = fork('modules/worker/export-worker.js');
+
+        forked.send(JSON.stringify({
+            datasetId,
+            standardId,
+            handlerId,
+            config: this.config,
+        }));
+
+        forked.on('message', async (response) => {
+            if (response.error) {
+                await this.handleError(handlerId, response.error);
+            } else {
+                this.logger.info(`Export complete for export handler_id: ${handlerId}`);
+
+                const handler = await Models.handler_ids.findOne({
+                    where: { handler_id: handlerId },
+                });
+                const data = JSON.parse(handler.data);
+
+                const cacheDirectory = path.join(this.config.appDataPath, 'export_cache');
+                const documentPath = path.join(cacheDirectory, handlerId);
+
+                const { dc_node_wallet, data_creator } = JSON.parse(fs.readFileSync(documentPath, { encoding: 'utf-8' }));
+                data.dc_node_wallet = dc_node_wallet;
+                data.data_creator = data_creator;
+
+                if (data.readExport) {
+                    data.export_status = 'COMPLETED';
+                    handler.status = data.import_status === 'COMPLETED' ? 'COMPLETED' : handler.status;
```

Comment:
I'm worried that this will get stuck in pending..
If the import becomes FAILED, and the export gets completed, the handler status won't get updated, and it will just stay PENDING.